libvast/src/plugin.cpp

```diff
@@ -145,19 +155,22 @@ plugin_ptr::~plugin_ptr() noexcept {
     dlclose(library_);
     library_ = {};
   }
+  version_ = {};
```

Comment:
Rework destructor logic for static plugins

Reply:
The logic was right. I confused `instance_` and `library_` in the peer-review when looking at this.